### DISCUSSION TITLE
Grant-PnPTenantServicePrincipalPermission : Access is denied

### DISCUSSION THREAD

**Discussion by Niko-F** [YannickRe/spfx-msgraph-peoplesearch#45] [Q&A] [answered]
Hi,

In debug window I see only my own filed, all other people has 'null' field.
I've tried to execute this 2 commands : 
- Connect-PnPOnline -url https://***mytenant***.sharepoint.com/ -UseWebLogin
- Grant-PnPTenantServicePrincipalPermission -Scope "User.Read.All" -Resource "Microsoft Graph" on my tenant

But the result is an error :
Grant-PnPTenantServicePrincipalPermission : Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
Au caractère Ligne:1 : 1
+ Grant-PnPTenantServicePrincipalPermission -Scope "User.Read.All" -Res ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Grant-PnPTenant...cipalPermission], ServerUnauthorizedAccessException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.Apps.GrantTenantServicePrincipalPermission

Could you help me ?

**Comment by YannickRe** [comment]
Did you login with a Global Administrator account? It's required to add these scopes.

**Reply by Niko-F** [reply, marked as answer]
Hi and thanks for your answer. Yes I did.
I've fixed the problem by adding the permission throught the graphical admin pannel.
